README.md

```diff
@@ -1,11 +1,21 @@
 # commercetools Postman collections
 
-This repository contains Postman collections for the commercetools platform.
+This repository contains Postman collections for the commercetools APIs.
+
+## Why use Postman collections?
+
+Postman collections help you explore commercetools APIs with a setup time of minutes. You can theoretically run the whole commerce journey from creating customers and products, to creating orders where these customers purchase the products.
+
+Key benefits of using Postman collections of commercetools APIs:
+* Works with minimum technical knowledge.
+* No coding is required, and can be used as soon as you have created a commercetools project.
+* Easily explore all the API endpoints and achieve a holistic view of how the platform and other services operate.
+
+[Get Started here](GettingStarted.md) with commercetools Postman collections.
 
 ## Disclaimer
 
-This is not the official commercetools platform documentation. Please see [here](http://docs.commercetools.com/)
-for a complete and approved documentation of the commercetools platform API.
+This is not the official commercetools platform documentation. To read complete documentation for the commercetools platform, please visit http://docs.commercetools.com/
```

Comment:
What's the use of this disclaimer? I would change this to a more friendly "to learn more about ..... visit documentation...."

Reply:
```suggestion
To learn more about using commercetools APIs, including full platform documentation, please visit https://docs.commercetools.com/
```